frontend/src/components/filters/FiltersInfoTS.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { FILTER_TYPE } from "../../constants";
+import { FilterDescription, FilterSet, FilterValue, RangeFilterValue, StringArrayFilterValue, isMetadataFilterValue, isRangeFilterValue, isStringArrayFilterValue } from "../../models/paged_items";
+
+interface FiltersInfosProps {
+    filters: FilterSet
+}
+
+const FiltersInfos = ({ filters }: FiltersInfosProps) => {
+    const appliedFilters = Object.keys(filters).filter(key => filters[key]?.value)
+    const getValue = (key: string) => {
+        const filterValue: FilterValue = filters[key].value ?? {}
+        const description: FilterDescription = filters[key].description ?? { type: '', key: '', label: '' }
+        let valueJSX;
+        if (description && filterValue) {
+            const descriptionJSX = (
+                <>
+                    {
+                        <strong>{description.label}: </strong>
+                    }
+                </>
+            );
+
+            switch (description.type) {
+                case FILTER_TYPE.SELECT: {
+                    if (isStringArrayFilterValue(filterValue)) {
```

Comment:
This needs to support both a string array filter value, and a single string filter value. Right now, the Depeleted filter value is being displayed as "true" or "false", but the old JS code displayed the labels "Yes" or "No". The value is a boolean, but we want to display the label to match the Yes or No labels the user sees in the filter control.